[2022-02-19 05:43:34,358] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T05:42:18.164074+00:00 [queued]>
[2022-02-19 05:43:34,369] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T05:42:18.164074+00:00 [queued]>
[2022-02-19 05:43:34,369] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 05:43:34,369] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 05:43:34,369] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 05:43:34,383] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): Load_Data.Save_to_DB> on 2022-02-19 05:42:18.164074+00:00
[2022-02-19 05:43:34,387] {standard_task_runner.py:52} INFO - Started process 15386 to run task
[2022-02-19 05:43:34,401] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'Load_Data.Save_to_DB', 'manual__2022-02-19T05:42:18.164074+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmp78d0ofri', '--error-file', '/tmp/tmp76epm3o8']
[2022-02-19 05:43:34,402] {standard_task_runner.py:77} INFO - Job 238: Subtask Load_Data.Save_to_DB
[2022-02-19 05:43:34,584] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T05:42:18.164074+00:00 [running]> on host c39e331ab017
[2022-02-19 05:43:34,674] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=Load_Data.Save_to_DB
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T05:42:18.164074+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T05:42:18.164074+00:00
[2022-02-19 05:43:34,691] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-02-19 05:43:34,693] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-02-19 05:43:34,694] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-02-19 05:43:34,694] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-02-19 05:43:34,695] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,696] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,697] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,698] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-02-19 05:43:34,699] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,701] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,701] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,702] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 05:43:34,757] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 05:43:34,793] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 05:43:34,796] {logging_mixin.py:109} INFO - Empty DataFrame
Columns: [VendorID, lpep_pickup_datetime, lpep_dropoff_datetime, RatecodeID, PULocationID, DOLocationID, passenger_count, trip_distance, fare_amount, extra, mta_tax, tip_amount, tolls_amount, improvement_surcharge, total_amount, payment_type, trip_type, congestion_surcharge]
Index: []
[2022-02-19 05:43:34,796] {logging_mixin.py:109} INFO - Save Data to Database Done !!
[2022-02-19 05:43:34,796] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 05:43:34,812] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=Load_Data.Save_to_DB, execution_date=20220219T054218, start_date=20220219T054334, end_date=20220219T054334
[2022-02-19 05:43:34,863] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 05:43:34,909] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
